CodesmithProjects/TrailBlazers
# Google OAuth2 login added

## Overview


Issue Type


Description
Users can now login to the site with their Google accounts.

Removed old API Key files and moved API Keys to a .env file.

Ticket Item

FEATURE#2: DB Spike #1

Steps to Validate Feature

1. Start the frontend and backend server
2. Go to 'http://127.0.0.1:5173'
3. Click on 'Login' at the top right of the nav bar
4. Choose your google account to login with
5. You should be redirected back to the home page. Address bar should now contain your OAuth access token.

Screenshots & Videos

<img width="1728" alt="Screenshot 2023-04-10 at 12 55 35 AM" src="https://user-images.githubusercontent.com/118780080/230829093-b6fec724-0547-4b63-9453-b28ee566f533.png">

<img width="457" alt="Screenshot 2023-04-10 at 12 57 43 AM" src="https://user-images.githubusercontent.com/118780080/230829133-81b67cd2-28f9-4837-9b8d-c479bdaf03c6.png">


Additional context
Will need to touch base with the group on creating the .env files to ensure team still has access to the API's in their own tests.
